influxdb_iox/tests/end_to_end_ng_cases/querier.rs

```diff
@@ -24,37 +19,31 @@ async fn basic_ingester() {
         .with_querier(querier_config)
         .await;
 
-    // Write some data into the v2 HTTP API ==============
-    let lp = format!(
-        "{},tag1=A,tag2=B val=42i 123456\n\
-                      {},tag1=A,tag2=C val=43i 123457",
-        table_name, table_name
-    );
-    let response = cluster.write_to_router(lp).await;
-    assert_eq!(response.status(), StatusCode::NO_CONTENT);
-
-    // Wait for data to be readable
-    let write_token = get_write_token(&response);
-    wait_for_readable(write_token, cluster.ingester().ingester_grpc_connection()).await;
-
-    // run query
-    let sql = format!("select * from {}", table_name);
-    let batches = run_query(
-        sql,
-        cluster.namespace(),
-        cluster.querier().querier_grpc_connection(),
+    StepTest::new(
+        &cluster,
+        vec![
+            Step::WriteLineProtocol(format!(
+                "{},tag1=A,tag2=B val=42i 123456\n\
+             {},tag1=A,tag2=C val=43i 123457",
```

Comment:
Nit: Maybe indent this second line so it lines up with the first line, to better see the differences in the data?

```suggestion
                "{},tag1=A,tag2=B val=42i 123456\n\
                 {},tag1=A,tag2=C val=43i 123457",
```

Reply:
fixed in 525634962